BankAccount/pkg/store/user.go

```diff
@@ -1 +1,75 @@
 package store
+
+import (
+	"bankacc/pkg/entities"
+	"database/sql"
+	"time"
+)
+
+type UserStore interface {
+	Insert(fullName string, email string, phoneNumber string, createdAt time.Time, updatedAt time.Time) (*entities.User, error)
```

Comment:
@F1c1cA you don't need to pass createdAt and updatedAt as arguments, they should be created when you execute the query.